src/en/release/1.7.1.md

```diff
@@ -0,0 +1,179 @@
+- [KYUUBI #4471] Prompted command for docs generation should skip Java tests
+- [KYUUBI #4482] Support to disable batch resource upload function
+- [KYUUBI #4452] Strip the redundant leading and tailing slash of getZooKeeperNamespace.
+- [KYUUBI #4451] Skip etcd test if no docker env
+- [KYUUBI #4433] Fix the case with `create a view from view`
+- [KYUUBI #4464] Simplify and improve log for JDBCMetadataStore
+- [KYUUBI #4462] Fix variable usage issue in `SessionManager#stop`
+- [KYUUBI #4443] Do not set engine session init sql for alive probe session
+- Spark Engine
+- [KYUUBI #4754] [ARROW] Use `KyuubiArrowConveters#toBatchIterator` instead of `ArrowConveters#toBatchIterator`
+- [KYUUBI #4749]  Fix flaky test issues in SchedulerPoolSuite
+- [KYUUBI #4722] [DOCS] Kyuubi requires Spark 3.1 and above now
+- [KYUUBI #4720] [ARROW] Fix java.lang.NoSuchFieldError: IpcOption.DEFAULT for Spark-3.1/3.2
+- [KYUUBI #4713][TEST] Fix false positive result in SchedulerPoolSuite (#4714)
+- [KYUUBI #4713][TEST] Fix false positive result in SchedulerPoolSuite
+- [KYUUBI #4710] [ARROW] LocalTableScanExec should not trigger job
+- [KYUUBI #4702] [ARROW] CommandResultExec should not trigger job
+- [KYUUBI #4662] [ARROW] Arrow serialization should not introduce extra shuffle for outermost limit
+- [KYUUBI #4587] [SPARK][EXT] Take care of table cache query stage
+- Spark Authz Plugin
+- [KYUUBI #4585] Authz policy file generation
+- [KYUUBI #4658] [Authz] [Bug] Fix InsertIntoHiveDirCommand classname so that we can extract the query in it when authorization.
+- [KYUUBI #4617] [AUTHZ] Collect results for filtered show objects ahead to prevent holding unserializable spark plan
+- [KYUUBI #4504] [Authz] [Bug] Fix source table privilege requirement when querying permanent view in Spark 3.1 and below
+- [KYUUBI #4532] [AUTHZ] Displays the columns involved in extracting the aggregation operator
+- [KYUUBI #4450] Ignore unknown fields `policyPriority` when reading policy json file
+- Other Spark Extensions
+- [KYUUBI #4727] [DOC] kyuubi-spark-lineage has no transitive deps
+- [KYUUBI #4586] [TEST][1.7] Fix to clean up views in lineage unit tests
+- [KYUUBI #4525][KSHC] Partitioning predicates should take effect to filter data
+- [KYUUBI #4331] [KYUUBI #4431] Lineage supports `Union` and `MergeInto`
+
+### Flink Engine
+- [KYUUBI #4666] Support Flink varbinary type in query operation
+- [KYUUBI #4635] Support Flink time type in query operation
+- [KYUUBI #4614] [DOCS] Rename Flink Table Store to Apache Paimon (Incubating) in docs `Connectors for Flink SQL Query Engine`
+- [KYUUBI #4607]  [DOCS] Rename Flink Table Store to Apache Paimon (Incubating) in docs `Connectors for Hive SQL Query Engine`
+- [KYUUBI #4608] [DOCS] Rename Flink Table Store to Apache Paimon (Incubating)  in docs `Connectors for Trino SQL Query Engine`
+- [KYUUBI #4598] Bump Flink-1.15 from 1.15.3 to 1.15.4
+- [KYUUBI #4583] [DOCS] Rename Flink Table Store to Apache Paimon (Incubating)
```

Comment:
We can remove the space between `[LABEL]`s

Reply:
Addressed.